Gems/LyShine/Code/Editor/SpriteBorderEditor.cpp

```diff
@@ -37,7 +37,7 @@ SpriteBorderEditor::SpriteBorderEditor(const char* path, QWidget* parent)
 
     // Make sure the sprite can load and be displayed before continuing
     m_sprite = AZ::Interface<ILyShine>::Get()->LoadSprite(m_spritePath.toLatin1().constData());
-    QString fullPath = Path::GamePathToFullPath(m_sprite->GetTexturePathname().c_str());
+    QString fullPath = Path::GamePathToFullPath((m_sprite->GetTexturePathname() + ".streamingimage").c_str());
```

Comment:
Please use `AZ::RPI::StreamingImageAsset::Extension` here and in the below instances so we don't have the `".streamingimage"` string duplicated in the codebase.

Reply:
done.